girder/utility/assetstore_utilities.py

```diff
@@ -60,6 +59,15 @@ def getAssetstoreAdapter(assetstore, instance=True):
         return cls
 
 
+def setAssetstoreAdapter(storeType, cls):
+    global assetstoreTable
```

Comment:
Good catch :-D

On Thu, Jun 9, 2016 at 2:56 PM Michael Fogleman <notifications@github.com>
wrote:

> In girder/utility/assetstore_utilities.py
> <https://github.com/girder/girder/pull/1372#discussion_r66500945>:
>
> > @@ -60,6 +59,15 @@ def getAssetstoreAdapter(assetstore, instance=True):
> >          return cls
> >
> >
> > +def setAssetstoreAdapter(storeType, cls):
> > +    global assetstoreTable
>
> This isn't needed unless you reassign the variable (which you aren't)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/girder/girder/pull/1372/files/91b74f1930d5a4c381b47090446bc4fccef8f68b#r66500945>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACxNJK2NtZWIWyKLeV13PjiN5cGk29Laks5qKGH3gaJpZM4IyQ3c>
> .
>